contrib/babelfishpg_tsql/sql/upgrades/babelfishpg_tsql--3.1.0--3.2.0.sql

```diff
@@ -2202,6 +2202,382 @@ GRANT SELECT ON sys.servers TO PUBLIC;
 CALL sys.babelfish_drop_deprecated_object('procedure', 'sys', 'babelfish_sp_rename_internal_deprecated_in_3_2_0');
 CALL sys.babelfish_drop_deprecated_object('procedure', 'sys', 'sp_rename_deprecated_in_3_2_0');
 
+ALTER TABLE sys.extended_properties RENAME TO extended_properties_deprecated_in_3_2_0;
```

Comment:
this would not be released in 3.2, 
I think you may create a new upgrade file from 3.2 -> 3.3

Reply:
done